spec/board_spec.rb

```diff
@@ -241,4 +241,45 @@
       expect(board.full?).to eq(false)
     end
   end
+
+  describe 'valid_move?' do
+    it 'returns true if when the user enters a valid number 1 -9 and the position is not taken' do
+      board = Board.new
+
+      board.mark(4)
+      board.mark(5, 'O')
```

Comment:
You don't really need `board.mark(4)` or `board.mark(5, 'O')` for this test. Remember, your tests should have just enough information/setup to make them work.